RELEASE.md

```diff
@@ -1,3 +1,32 @@
+# Release 1.0.0
+
+## Breaking Changes
+
+- New parameter `tensor_fields` will replace `non_tensor_fields` in the `add_documents` endpoint. Only fields in `tensor_fields` will have embeddings generated, thereby offering more granular control as to which fields to vectorise. The `non_tensor_fields` parameter is deprecated and will be removed in a future release. Calls to `add_documents` with neither of these parameters specified will now fail [#538](https://github.com/marqo-ai/marqo/pull/538).
+
+- Single KNN Vector Optimisation. Only one OpenSearch KNN vector field will be created per index, instead of one per field. This results in faster and more stable searches across multiple tensor fields. Note that indexed documents will now have a different OpenSearch document structure, so documents indexed with previous Marqo versions cannot be searched with this version, and vice versa [#530](https://github.com/marqo-ai/marqo/pull/530).
+
+- The `add documents` endpoint's request body is now an object, with the list of documents under the `documents` key. The query parameters `use_existing_tensors`, `image_download_headers`, `model_auth`, and `mappings` have also been moved to the body as optional keys, and support for these parameters in the query string is deprecated. Backwards compatibility is supported at the moment, but will be removed in a later release. This change results in better shorter URLs as well as better readability, since these parameters are no longer URL-encoded [#535](https://github.com/marqo-ai/marqo/pull/535).
```

Comment:
You may also want a link to the doc for this one so bash curl user know what is the correct API now.